src/translations/en.json

```diff
@@ -511,6 +511,9 @@
   "NoAccessPage.postListings.schemaTitle": "No publishing rights",
   "NoAccessPage.postListings.heading": "You can't publish listings",
   "NoAccessPage.postListings.content": "You need to receive publishing rights from the {marketplaceName} team.",
+  "NoAccessPage.initiateTransactions.schemaTitle": "No transaction rights",
+  "NoAccessPage.initiateTransactions.heading": "You can't initiate transactions",
+  "NoAccessPage.initiateTransactions.content": "You need to receive transaction rights from the {marketplaceName} team.",
```

Comment:
Btw. were these approved by Katri (I think she has lately been the one proofreading the translations)

Reply:
These texts were from the most recent wireframes, but I'll double check!